backend/yugioh/models.py

```diff
@@ -0,0 +1,118 @@
+from django.db import models
+
+
+class YugiohCard(models.Model):
+
+    type = models.CharField(
+        max_length=100,
+    )
+
+    frame_type = models.CharField(
+        max_length=100,
+    )
+
+    description = models.CharField(
+        max_length=1000,
+        blank=True,
+        null=True,
+    )
+
+    attack = models.CharField(
+        max_length=100,
+        blank=True,
+        null=True,
+    )
+
+    defense = models.CharField(
+        max_length=100,
+        blank=True,
+        null=True,
+    )
+
+    level = models.CharField(
+        max_length=100,
+        blank=True,
+        null=True,
+    )
+
+    race = models.CharField(
+        max_length=100,
+    )
+
+    attribute = models.CharField(
+        max_length=100,
+        blank=True,
+        null=True,
+    )
+
+    archetype = models.CharField(
+        max_length=100,
+        blank=True,
+        null=True,
+    )
+
+    image = models.URLField(
+        blank=True,
+        null=True,
+    )
+
+    card = models.ForeignKey(
+        'card.Card',
+        on_delete=models.CASCADE,
+    )
+
+    def __str__(self):
+        return self.card.card_name
+
+
+class YugiohCardInSet(models.Model):
+    rarity = models.ForeignKey(
+        'YugiohCardRarity',
+        on_delete=models.CASCADE,
+    )
+    set = models.ForeignKey(
+        'YugiohCardSet',
+        on_delete=models.CASCADE,
+    )
+
+    yugioh_card = models.ForeignKey(
+        YugiohCard,
+        on_delete=models.CASCADE,
+    )
+
+    def __str__(self):
+        return f'{self.set} - {self.rarity}'
```

Comment:
- [ ] Represent as `card name` - `set` - `rarity`

Reply:
The new represent was set.